### PR TITLE
Fix monitor not reposting

### PR DESCRIPTION
**Pull request by Mygod**
rude

### CODE REVIEW

File: src/services/utils.ts [hunk 1]
```diff
@@ -73,9 +73,11 @@ export function buildProxy(
 }
 
 export function getEmbed(status: ProxyStatus, proxy: string): APIEmbed {
+  let color: number = Colors.Red
+  if (status.code === 200) color = Colors.Green; else if (status.code > 200 && status.code < 400) color = Colors.Yellow
```
Comment (TurtIeSocks) [inline src/services/utils.ts]: Can you rewrite this so it follows the codebase style and is clearer to read?

File: src/services/utils.ts [hunk 2]
```diff
@@ -73,9 +73,11 @@ export function buildProxy(
...
   return {
     title: `${status.code} for ${proxy} Proxy`,
-    color: { 200: Colors.Green, 500: Colors.Red }[status.code] || Colors.Yellow,
+    color: color,
```
Comment (TurtIeSocks) [inline src/services/utils.ts]: Please use the simplified version since the variable name is the same as the key. e.g. `{ color: color }` => `{ color }`

**Review by TurtIeSocks — CHANGES_REQUESTED** [review]
Going to test the logic this afternoon regarding sending new messages vs editing but please make the small changes in the meantime.

File: src/services/utils.ts (hunk 2 above)
Comment (Mygod) [reply src/services/utils.ts]: Long time no write js. These changes lgtm.

**Review by TurtIeSocks — APPROVED** [review]

**Pull request merged**